conf/apisix/apisix.yaml

```diff
@@ -337,6 +337,143 @@ routes:
             # Fix CWE Id: 693
             X-Content-Type-Options: nosniff
+  -
+    uri: /fam-wui
+    plugins:
+      redirect:
+        uri: /fam-wui/
+      response-rewrite:
+        headers: 
+          set:
+            # Fix CWE Id: 693
+            Content-Security-Policy: "default-src https: data: blob: 'self'; connect-src 'self' data: blob:; media-src 'self'; style-src 'self' 'unsafe-inline'; script-src 'self' 'unsafe-inline' 'unsafe-eval' data: blob:; font-src 'self'; img-src 'self' data: blob:; frame-ancestors 'none'; form-action 'self'; object-src 'none'; manifest-src 'self'; frame-src 'self'"
+            # Fix CWE Id: 200
+            Server: "ARLAS services"
+            # Fix CWE Id: 319 // Value recommended by https://hstspreload.org/
+            Strict-Transport-Security: max-age=63072000; includeSubDomains; preload
+            # Fix CWE Id: 693
+            X-Content-Type-Options: nosniff
+  -
+    uri: /fam-wui/*
+    upstream:
+      nodes:
+        "arlas-fam-wui:80": 1
+    plugins:
+      response-rewrite:
+        headers: 
+          set:
+            # Fix CWE Id: 693
+            Content-Security-Policy: "default-src https: data: blob: 'self'; connect-src 'self' data: blob:; media-src 'self'; style-src 'self' 'unsafe-inline'; script-src 'self' 'unsafe-inline' 'unsafe-eval' data: blob:; font-src 'self'; img-src 'self' data: blob:; frame-ancestors 'none'; form-action 'self'; object-src 'none'; manifest-src 'self'; frame-src 'self'"
+            # Fix CWE Id: 1021
+            X-Frame-Options: "DENY"
+            # Fix CWE Id: 693 // The () means those resources are not allowed.
+            Permissions-Policy: accelerometer=(), camera=(), microphone=(), geolocation=(), usb=()
+            # Fix CWE Id: 200
+            Server: "ARLAS services"
+            # Fix CWE Id: 319 // Value recommended by https://hstspreload.org/
+            Strict-Transport-Security: max-age=63072000; includeSubDomains; preload
+            # Fix CWE Id: 693
+            X-Content-Type-Options: nosniff
+      proxy-rewrite:
+        regex_uri: ["/fam-wui/(.*)", "/$1"]
+  -
+    uri: /agate/healthcheck
+    methods: ["GET"]
+    upstream:
+      nodes:
+        "agate:8004": 1
+  -
+    uri: /airs-storage/*
+    methods: ["GET"]
+    upstream:
+      nodes:
+        "minio:9000": 1
+    plugins:
+      forward-auth:
+        uri: http://agate:8004/agate/authorization
+        request_headers: ["Authorization", "arlas-org-filter"]
+  -
+    uri: /fam/*
+    upstream:
+      nodes:
+        "fam-service:8005": 1
+    plugins:
+      forward-auth:
+        uri: http://arlas-iam-server:9998/arlas_iam_server/auth
+        request_headers: ["Authorization", "arlas-org-filter"]
+  -
+    uri: /fam/files
+    upstream:
+      nodes:
+        "fam-service:8005": 1
+    plugins:
+      forward-auth:
+        uri: http://arlas-iam-server:9998/arlas_iam_server/auth
+        request_headers: ["Authorization", "arlas-org-filter"]
+      # proxy-rewrite:
```

Comment:
to remove ?

Reply:
I'll do a pass after this merge to optimize this # proxy-rewrite: part